Windows-Service.md

```diff
@@ -104,16 +109,21 @@ goto printUsage
 echo Insert the OrientDB Home
 goto printUsage
 
+:missingRootPassword
+echo Insert the root password
+goto printUsage
+
 :printUsage
 echo usage:
-echo     installService JVM_DLL_location OrientDB_Home
+echo     installService JVM_DLL_location OrientDB_Home root_password
 EXIT /B
 ```
 
 The script requires two input parameters:
```

Comment:
I suppose you missed small mistake, script requires three input parameters ))

Reply:
Fixed :)